app/views/shared/_services-nav.html.erb

```diff
@@ -1,50 +1,108 @@
-        <%= link_to "All", admin_services_path, class: "shortcuts__link" %>
-    <% else %>
-        <span class="shortcuts__current">All</span>
-    <% end %>
-
-    <span class="shortcuts__count">(<%= @all_count %>)</span>
-
-    <span class="shortcuts__divider">|</span>
-
-    <% if APP_CONFIG["directories"].present? %>
-        <% APP_CONFIG["directories"].each do |directory| %>
-            <% if (params[:directory] === directory["value"]) %>
-                <span class="shortcuts__current"><%= directory["value"] %></span>
-            <% else %>
-                <span class="shortcuts__link"><%= link_to directory["value"], admin_services_path(directory: directory["value"])%></span>
-            <% end %>
-            <span class="shortcuts__count">(<%= @directories_counts[directory["name"]] %>)</span>
-            <span class="shortcuts__divider">|</span>
-        <% end %>
-    <% end %>
-
-    <% if (params[:ofsted] === "true") %>
-        <span class="shortcuts__current">Ofsted registered</span>
-    <% else %>
-        <%= link_to "Ofsted registered", admin_services_path(ofsted: true), class: "shortcuts__link" %>
-    <% end %>
-
-    <span class="shortcuts__count">(<%= @ofsted_count %>)</span>
-
-    <span class="shortcuts__divider">|</span>
-
-    <%= link_to_unless_current "Pending", admin_requests_path, class: "shortcuts__link" do %>
-        <span class="shortcuts__current">Pending</span>
-    <% end %>
-    <span class="shortcuts__count">(<%= @pending_count %>)</span>
-
-    <span class="shortcuts__divider">|</span>
-
-    <% if (params[:archived] === "true") %>
-        <span class="shortcuts__current">Archived</span>
-    <% else %>
-        <%= link_to "Archived", admin_services_path(archived: true), class: "shortcuts__link" %>
-    <% end %>
-
-    <span class="shortcuts__count">(<%= @archived_count %>)</span>
+  <% 
+    @directory_links = {}
+
+    if APP_CONFIG["directories"].present?
+      APP_CONFIG["directories"].each do |directory|
+        @directory_links[directory["value"]] = admin_services_path(directory: directory["value"]);
+      end
+    end
+
+%>
+  
+  <%= 
+  # display the bfis/bod headings
+  render partial: "shared/directory-shortcuts",  locals: { 
+        all_state: !params[:directory].present?,
+        all_link: admin_services_path,
+        all_count: @service_counts[:all][:all],
+        all_label: "All",
+        directory_links: @directory_links,
+        directory_counts: @service_counts
+  }
+
+  #directory_link: admin_services_path(directory: directory["value"])
+%>
+
+
+
+<nav class="shortcuts shortcuts--no-top-spacing  shortcuts--sublevel">
+
+<% 
+
+
+    @sublevel_ofsted_state = false;
+    @sublevel_pending_state = false;
+    @sublevel_archived_state = false;
+
+    # toggle the active state for the ofsted filter
+    if params[:ofsted].present? && params[:ofsted] === "true" 
+        @sublevel_ofsted_state = true;
+    else    
+        @sublevel_ofsted_state = false;
+    end
```

Comment:
The conditions we're checking evaluate to true/false, so could these instead be:

```
@sublevel_ofsted_state = params[:ofsted] === "true"
@sublevel_pending_state = current_page?(controller: 'admin/requests', action: 'index')
...
```

You can leave out the presence check because if it's not present, the comparison will evaluate to false :)

Reply:
have updated the refactored logic with these changes 👍